packages/@ourworldindata/grapher/src/controls/Controls.tsx

```diff
@@ -208,47 +649,59 @@ export class FacetStrategyDropdown extends React.Component<{
         )
     }
 
-    // A hovering visual display giving options to be selected from
-    @computed get content(): JSX.Element {
-        const parts = this.strategies.map((value: FacetStrategy) => {
-            const label = this.facetStrategyLabels[value]
-            const children =
-                value === FacetStrategy.none ? (
-                    // a single solid block
-                    <div className="FacetStrategyPreview-none-child"></div>
-                ) : (
-                    // a 3x2 grid of squares
-                    <>
-                        <div className="FacetStrategyPreview-split-child"></div>
-                        <div className="FacetStrategyPreview-split-child"></div>
-                        <div className="FacetStrategyPreview-split-child"></div>
-                        <div className="FacetStrategyPreview-split-child"></div>
-                        <div className="FacetStrategyPreview-split-child"></div>
-                        <div className="FacetStrategyPreview-split-child"></div>
-                    </>
-                )
+    @computed get subtitle(): string {
+        const entityType = this.props.manager.entityType ?? "country or region",
+            byEntity = this.strategies.includes(FacetStrategy.entity),
+            byMetric = this.strategies.includes(FacetStrategy.metric)
+
+        if (byEntity || byMetric) {
+            const facet =
+                byEntity && byMetric
+                    ? `metric, ${entityType}`
+                    : byEntity
+                    ? entityType
+                    : "metric"
```

Comment:
We could also use the manager's `facettingLabelByYVariables` here if we wanted to. `facettingLabelByYVariables` is an alternative for "metric", specified by the author of a chart.